trackalert/trackalert-lua.cc

```diff
@@ -297,6 +297,21 @@ vector<std::function<void(void)>> setupLua(bool client, bool multi_lua,
     c_lua.writeFunction("showVersion", []() { });
   }
 
+  if (!multi_lua) {
```

Comment:
A bit confusing, `g_key` was being abandoned but is still used here???

Reply:
That's because the trackalert daemon doesn't use the WforceReplication class - it only uses encryption for the console, hence still using g_key.